Makefile

```diff
@@ -172,7 +172,8 @@ release-build:
 	cd ../reana && \
 	echo "The following images have been built:" && \
 	./scripts/update_images.sh $$BUILT_IMAGES_FILE_PATH && \
-	rm $$BUILT_IMAGES_FILE_PATH
+	rm $$BUILT_IMAGES_FILE_PATH && \
+	pip install ../reana-client --upgrade
```

Comment:
Hmm, is single `pip` call enough? Do we need this? If so, we should probably pip uninstall a bunch, as we do in the dev build?  Anyway, we could also address this as part of #304.